backend/src/hmc-api/course.ts

```diff
@@ -492,18 +493,21 @@ function processCourseAreas(
     }
 }
 
-export function linkCourseData(files: {
-    altstaff: string;
-    calendarSession: string;
-    calendarSessionSection: string;
-    course: string;
-    courseSection: string;
-    courseSectionSchedule: string;
-    permCount: string;
-    sectionInstructor: string;
-    staff: string;
-    courseAreas: string;
-}): APIv4.Section[] {
+export const CourseFiles = z.object({
```

Comment:
what's the mapping here? mapping from the name we're using to refer to the file in codebase to the literal csv file name? can we add a comment/docstring explaining that?